test/unit/presenters/formats/edition_format_presenter_test.rb

```diff
@@ -58,6 +58,10 @@ def result
       assert_equal "override me", result[:schema_name]
     end
 
+    should "[:rendering_app]" do
```

Comment:
This is still [duplicated](https://github.com/alphagov/publisher/blob/e053089c496a6074c54cd30163561f4be84717c6/test/unit/presenters/formats/edition_format_presenter_test.rb#L106) in this class